pkg/virt-launcher/virtwrap/network/podnic_test.go

```diff
@@ -198,6 +195,116 @@ var _ = Describe("podNIC", func() {
 			})
 		})
 	})
+	type stateRetrievalTestCase struct {
+		storedState   *cache.PodIfaceState
+		expectedState cache.PodIfaceState
+	}
+	ptr := func(state cache.PodIfaceState) *cache.PodIfaceState {
+		return &state
+	}
+	DescribeTable("state is retrieved", func(c stateRetrievalTestCase) {
```

Comment:
this is for sure not crucial, but why pass a struct and not just pass storedState and expectedState as the case arguments?

Reply:
Using an argument is a way to put clear names to the arguments at every `Entry`.

Reply:
yes, I thought the names will be `expectedState` and  `storedState` instead of `c.expectedState` and  `c.storedState`

Reply:
Those names will be like that inside the function from a caller perspective they are not enforced.